.gitpod.yml

```diff
@@ -1,8 +1,8 @@
 ports:
 - port: 3000
 tasks:
-- init: yarn install
-  command: cd examples/browser && yarn start --hostname 0.0.0.0 ../..
+- init: nvm install 10 && nvm use 10 && yarn
```

Comment:
Why do not we install `10.15.3` if the documentation says: _"Preferably, **use** version `10.15.3`"_

Reply:
what issues do you see with using latest node 10? before we were using latest node 8, seemed to work just fine

Reply:
it's the same in travis config btw

Reply:
I just do not understand why do we recommend a version in the documentation but do not stick to it in the gitpod script.

Reply:
Electron ships with Node 10.15.3, there is a slight danger you could introduce incompatibilities if you build using a newer version, however I wouldn't want to restrict people to a specific release otherwise.

This was the same with Electron 2, you can't override the node version it uses.

IMO, the current set up gives the user the right amount of flexibility.